src/components/v5/common/ActionSidebar/partials/UserSelect/hooks.ts

```diff
@@ -18,11 +43,21 @@ export const useUserSelect = (): UserSelectHookProps => {
     };
   });
 
+  const isRecipientNotVerified: boolean = recipient && !isUserVerified;
+
+  const userFormat: string =
+    isHexString(recipient) && recipient
+      ? splitWalletAddress(recipient)
+      : recipient;
+
   return {
     loading,
     options: users || [],
     key: 'users',
     title: { id: 'actions.recipent' },
     isAccordion: false,
+    isUserVerified,
+    isRecipientNotVerified,
+    userFormat,
```

Comment:
These things should be tied together with this
![image](https://github.com/JoinColony/colonyCDapp/assets/38806520/1a63f016-2645-4987-bb03-5bd703cebbce)
in a single hook as they are related to a selected user not to a list of options

Reply:
I mean you can put this logic in the same hook no?